packages/core/base/src/services/useCrossmintEvents.ts

```diff
@@ -1,23 +1,62 @@
 import { CheckoutEvents, ListenToMintingEventsProps, ListenerType } from "../models/events";
-import { fetchOrderStatus } from "../utils/payments";
+import { CrossmintCheckoutEvent } from "../models/paymentElement";
+import { getEnvironmentBaseUrl } from "../utils";
 
-export default function useCrossmintEvents() {
+export function useCrossmintEvents({ environment }: { environment?: string } = {}) {
```

Comment:
`CrossmintEnvironment`

Reply:
hmmm, left environment for consistency, since this is the name of the prop everywhere else